src/tridiag.jl

```diff
@@ -474,6 +474,19 @@ end
     end
 end
 
+@inline function getindex(A::SymTridiagonal, b::BandIndex)
+    @boundscheck checkbounds(A, i, j)
```

Comment:
The `i, j` reference here is wrong. Can one do directly

```suggestion
    @boundscheck checkbounds(A, b)
```

?

Reply:
Thanks for spotting! that should work